src/gui/wallet.go

```diff
@@ -113,62 +104,105 @@ func NewWalletRPC(walletDir string, options ...wallet.Option) *WalletRPC {
 	return rpc
 }
 
+func (wrpc *WalletRPC) removeDup(wlts wallet.Wallets) wallet.Wallets {
+	var dupWltIDS []string
+	// remove dup wallets
+	for wltID, wlt := range wlts {
+		addr := wlt.Entries[0].Address.String()
```

Comment:
I will add `len(Entries)` checking, in case the wallet is empty. But that will only occur if user delete the entries in the wallet json file manually, new created wallet will have a default entry. It should be safe to compare the first address to detect dup wallet.